runtime/parachains/src/mock.rs

```diff
@@ -296,6 +304,7 @@ impl inclusion::RewardValidators for TestRewardValidators {
 
 /// Create a new set of test externalities.
 pub fn new_test_ext(state: MockGenesisConfig) -> TestExternalities {
+	sp_tracing::try_init_simple();
```

Comment:
Guess this is a debugging artifact, is it still desirable to leave it included?

Reply:
Yes, I always prefer the _option_ to enable logging when running tests, which this line enables. It won't have any impact if you don't.